_data/apis.json

```diff
@@ -14866,5 +14866,842 @@
             "zh_cn",
             "zh_tw"
         ]
+    },
+    {
+        "name": "OpenAI GPT",
+        "id": "openai-gpt",
+        "company": "openai",
+        "urls": [
+            "https://platform.openai.com/docs/guides/gpt",
+            "https://platform.openai.com/docs/guides/prompt-engineering"
+        ],
+        "languages": [
+            "af",
+            "ar",
+            "az",
+            "be",
+            "bg",
+            "bs",
+            "ca",
+            "cs",
+            "cy",
+            "da",
+            "de",
+            "el",
+            "en",
+            "es",
+            "et",
+            "fa",
+            "fi",
+            "fr",
+            "gl",
+            "he",
+            "hi",
+            "hu",
+            "hy",
+            "id",
+            "is",
+            "it",
+            "ja",
+            "kn",
+            "kk",
+            "ko",
+            "lv",
+            "lt",
+            "mk",
+            "mi",
+            "mr",
+            "ms",
+            "ne",
+            "nl",
+            "no",
+            "pl",
+            "pt",
+            "ro",
+            "ru",
+            "sk",
+            "sl",
+            "sr",
+            "sv",
+            "sw",
+            "ta",
+            "th",
+            "tl",
+            "tr",
+            "uk",
+            "ur",
+            "vi",
+            "zh"
+        ],
+        "text_translation": true,
+        "speech_translation": true,
+        "file_translation": true,
+        "self-serve": true,
+        "type": "gen_ai",
+        "access_method": "model"
```

Comment:
I don't love this, because:
- The traditional APIs also all have underlying models.  They're just less transparent about it.
- All of them are accessed via an API.  It's just that these ones require additional prompting (to be passed to the API)
- We don't want to be forced to go add `"access_method": "api"` to all of the APIs we already covered.